contextual_lenses/pfam_family_classification/pfam_nearest_neighbors_sample_sweep.py

```diff
@@ -0,0 +1,126 @@
+"""Nearest neighbors classification sampling sweep
```

Comment:
migrating this file would be a great first step when migrating to caliban. Rather than doing this for loop in a single process, have each be it's own process in the cluster.

Reply:
agreed, working on that now